src/style.css

```diff
@@ -0,0 +1,307 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #f0f0f0;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  display: flex;
+  flex-direction: column;
+  color: #222369;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+a:hover {
+  color: #e1ddf0;
+}
+
+header {
+  background-color: #5442a2;
+}
+
+nav {
+  padding: 1.5rem 12%;
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+}
+
+nav ul {
+  display: flex;
+  list-style: none;
+}
+
+ul li {
+  margin-left: 40px;
+}
+
+.card-counter {
+  margin: 0 auto;
+  padding: 5rem 1% 2rem 1%;
+  text-align: center;
+}
+
+main {
+  margin: 0 15% 7rem 15%;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 5%;
+}
```

Comment:
- [ ] [OPTIONAL] the project looks great in Chrome, however, in Firefox there is a very big gap between the cards as you can see in the below print screen. this is due to the 5% gap on line 54. because it takes 5% of the entire length of the page.
![Zrzut ekranu z 2022-01-07 o 12 04 47](https://user-images.githubusercontent.com/39240820/148536278-024f1ec1-ad36-4cca-8925-12ed128bc05b.png)
If you want your project to be accessible to more users please consider adjusting the project so it looks as intended in other browsers as well.